src/components/Square.js

```diff
@@ -1,19 +1,15 @@
-import React from 'react';
+import React, {useState} from 'react';
```

Comment:
since you are not initializing a piece of state in this component, you don't need to import `useState`

```suggestion
import React from 'react';
```